src/components/TemplatedCaseStudy/FirstTextSection.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+
+import { BodyPrimary } from '../Typography'
+import { Row, Col } from '../grid'
+
+const RowLayout = styled(Row)`
+  ${breakpoint('smallTablet')`
+    justify-content: flex-end;
+  `}
+`
+
+const FirstTextSection = ({ textAboveVideo }) => (
```

Comment:
just a matter of style, I guess, but it seems `textAboveVideo` could be simply called `text` since this compo does not have the notion of any video (that's on its parent responsibility). No problem to keep it like this though :)

Reply:
Nah, I totally agree. Thanks :+1:

Reply:
Lemme know what you think of the changes

Reply:
I like them, you got my blessing!